WordPress/gutenberg
# Mapping Contextual Commands: Templates

Part of https://github.com/WordPress/gutenberg/issues/50407

## What?
Command that will be suggested and prioritized when the Command Center is invoked from specific flows (Site Editor)

## Testing Instructions
1. Open the site editor
2. Open the Command Palette (by pressing Cmd + K on Mac).
3. Confirm that the command prompt for "Templates" is showing without any search

## Screenshots or screencast 
<img width="1433" alt="templates-context" src="https://github.com/user-attachments/assets/0cabb2e2-1b3e-484c-a54b-c387bd92a844">
